### DISCUSSION TITLE
Wind stress instead of wind speed?

### DISCUSSION THREAD

**Discussion by AndresSepulveda** [OpenDrift/opendrift#1172] [Ideas]
Hi, 

We want to force a plastic model with wind and currents. Our ocean model, CROCO, can store in its variables the wind stress used to force the currents.

Could we use this info as wind forcing, instead of adding a reader for an atmospheric model?

**Comment by knutfrode** [comment]
In principle one could implement an `environment mapping method` to calculate wind speed from wind stress, given some assumptions.
https://github.com/OpenDrift/opendrift/blob/master/opendrift/readers/basereader/variables.py#L484

**Reply by Sabarrera19** [reply]
Hello, I'm using Sedimentdrift and I'm trying to calculate wind speed from wind stress, but I'm not sure how to implement an "environment mapping method." Could you please explain a little bit about it?
